integration/commands_administration_test.go

```diff
@@ -1046,9 +1046,11 @@ func TestCommandsAdministrationWhatsMyURI(t *testing.T) {
 	databaseName := s.Collection.Database().Name()
 	collectionName := s.Collection.Name()
 
+	// only check port number on TCP connection, no need to check on unix socket
+	isTCP := s.IsTPC(t)
```

Comment:
I believe there is a typo in `TCP`.

Reply:
Yes :sweat_smile:  thanks for catching this :pray: